pages/docs/concepts/asyncapi-document/structure.md

```diff
@@ -0,0 +1,429 @@
+---
```

Comment:
You did an excellent job breaking down these complex concepts into manageable chunks of information, @TRohit20! Just a few notes...  ✌🏽

----

1. **Description of the `servers` object:** I noticed that the `protocolVersion`, `title`, `summary`, `security`, `tags`, `externalDocs`, and `bindings` fields are mentioned in the textual explanation, but they are not present in the visual representation (Mermaid graph) or the code example of the `server` object. You might want to update the Mermaid graph and the code example to include them. 

2. **Consistency in the code examples:** You've used a single server as an example in the code example for the `servers` object. In contrast, the code example for the `channels` object provides a more comprehensive example that includes multiple fields. You might also want to provide a more detailed code example to enhance the clarity of the `servers` object. 

3. **Spelling Inconsistency in the `components` Mermaid graph:** The graph capitalizes `Category` and `Tag`, but that seems like a spelling mistake. 

4. **Inconsistent spelling in headers:** Remember that we capitalize our headers in _sentence_ case, not every word. Please correct your headers that do not observe sentence capitalization.